src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,48 @@
 package core.basesyntax;
 
-class Pair {
+public class Pair<F, S> {
+    private F first;
+    private S second;
 
+    private Pair(F first, S second) {
+        this.first = first;
+        this.second = second;
+    }
+
+    public F getFirst() {
+        return first;
+    }
+
+    public S getSecond() {
+        return second;
+    }
+
+    public static <F, S> Pair<F, S> of(F first, S second) {
+        return new Pair<F, S>(first, second);
```

Comment:
writing just `<>` here is possible because of the [diamond operator](https://www.baeldung.com/java-diamond-operator)
```suggestion
        return new Pair<>(first, second);
```